docs/release_notes.rst

```diff
@@ -20,12 +20,18 @@ This is a major release and contains breaking changes. Please read the notes bel
 * Python 3.6 is no longer supported.
 * :meth:`Index.count <pynamodb.indexes.Index.count>`, :meth:`Index.query <pynamodb.indexes.Index.query>`,
   and :meth:`Indexn.scan <pynamodb.indexes.Index.scan>` are now instance methods.
+* Switched back to botocore _make_api_call  under the hood to enable tracing with opentelemetry
```

Comment:
Let me see how to phrase it. There's multiple benefits, not the least that botocore changes kept breaking the library.